index.html

```diff
@@ -56,6 +57,264 @@ <h2 class="connect">LET'S CONNECT</h2>
           </ul>
         </div>
       </section>
+
+      <section id="works" class="work-section">
+        <div class="section-grid">
+
+          <div class="project-card">
+            <a href="#snapshot">
+              <img
+              class="snapshot-portfolio-img"
+                src="images/Snapshoot Portfolio.png"
+                alt="snapshot-portfolio"
+              />
+            </a>
+
+            <div class="left-block">
+              <h2 class="project-title">Tonic</h2>
+                <div class="counter">
+                  <h3 class="canopy">CANOPY</h3>
+                  <ul class="canopy-text">
+                    <li><img src="images/Counter.png" alt="circle" /></li>
+                    <li>Back End Dev</li>
+                    <li><img src="images/Counter.png-2.png" alt="circle" /></li>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
- The background color of the work section is not the same as the background color of each project card as you can see from the Figma design

| Your Design(background-color: #fff;) |Figma Design(background-color: #E5E5E5;) |
| :---:   | :---: | 
|![Screenshot from 2022-12-06 16-30-15](https://user-images.githubusercontent.com/92303891/205968301-6909c20c-4938-4e9f-9179-93d645751a81.png) | ![Screenshot from 2022-12-06 16-25-55](https://user-images.githubusercontent.com/92303891/205968127-cdb85ffa-df0b-4b2c-a0d4-4ad4f5b39c1b.png)|